### ISSUE TITLE
Tables are hard to read on mobile

### ISSUE THREAD

**Issue by macintoshhelper** [FACN1/week6-hibas-angels#41]
Maybe overflow: scroll could be used on the table or something. The text is hard to read in the columns.

**Comment by jwld** [comment]
Good point. We changed from a list layout to a table at the last minute because we had no CSS. We'd probably change it back to a styled list if we had time.
